<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DACFWidget</class>
 <widget class="QWidget" name="DACFWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>258</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DACFWidget</string>
  </property>
  <widget class="QPushButton" name="butSave">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>340</y>
     <width>110</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid gray;
</string>
   </property>
   <property name="text">
    <string>SAVE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="butExit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>340</y>
     <width>110</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid gray;
</string>
   </property>
   <property name="text">
    <string>EXIT</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmgRanges">
   <property name="geometry">
    <rect>
     <x>131</x>
     <y>20</y>
     <width>110</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox{
border:1px solid gray;
 }</string>
   </property>
   <item>
    <property name="text">
     <string>0.6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>14</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="lblRanges">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>110</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid gray;</string>
   </property>
   <property name="text">
    <string>Select Range :</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblOffset">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>110</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid gray;</string>
   </property>
   <property name="text">
    <string>Offset Value:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leOffset">
   <property name="geometry">
    <rect>
     <x>131</x>
     <y>70</y>
     <width>110</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid gray;
</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>194</width>
     <height>186</height>
    </rect>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>BlankCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid gray;
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="tabKeyNavigation">
    <bool>false</bool>
   </property>
   <property name="showDropIndicator" stdset="0">
    <bool>true</bool>
   </property>
   <property name="gridStyle">
    <enum>Qt::DotLine</enum>
   </property>
   <row>
    <property name="text">
     <string>0.6     </string>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>14</pointsize>
     </font>
    </property>
   </row>
   <row>
    <property name="text">
     <string>2</string>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>14</pointsize>
     </font>
    </property>
   </row>
   <row>
    <property name="text">
     <string>4</string>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>14</pointsize>
     </font>
    </property>
   </row>
   <row>
    <property name="text">
     <string>8</string>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>14</pointsize>
     </font>
    </property>
   </row>
   <row>
    <property name="text">
     <string>14</string>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>14</pointsize>
     </font>
    </property>
   </row>
   <column>
    <property name="text">
     <string>  Offset</string>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>14</pointsize>
     </font>
    </property>
   </column>
   <item row="0" column="0">
    <property name="text">
     <string/>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>14</pointsize>
     </font>
    </property>
   </item>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>5</width>
     <height>400</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:black;
background-color: rgb(0, 0, 0);
border-color: rgb(0, 0, 0);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>253</x>
     <y>0</y>
     <width>5</width>
     <height>400</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:black;
background-color: rgb(0, 0, 0);
border-color: rgb(0, 0, 0);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>395</y>
     <width>253</width>
     <height>5</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:black;
background-color: rgb(0, 0, 0);
border-color: rgb(0, 0, 0);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>0</y>
     <width>253</width>
     <height>5</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:black;
background-color: rgb(0, 0, 0);
border-color: rgb(0, 0, 0);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
